inc/fields/class-shortcode-ui-field-post-select.php

```diff
@@ -137,11 +137,21 @@ public function action_wp_ajax_shortcode_ui_post_field() {
 
 		$query = new WP_Query( $query_args );
 
+		$is_multiple_post_types = count( $query->get( 'post_type' ) ) > 1;
+
 		foreach ( $query->posts as $post_id ) {
+			$post_type     = get_post_type( $post_id );
+			$post_type_obj = get_post_type_object( $post_type);
```

Comment:
This line is triggering a code sniff violation: `Expected 1 spaces before closing bracket`.